tests/tasks/get_insights_tags.yml

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Get state of insights tag file
+  stat:
+    path: /etc/insights-client/tags.yml
+  register: __tags_file
```

Comment:
I think you can skip the extra variable, and directly register `test_insights_tags` (only a minor thing)

Reply:
You think right :smile: